Orange/data/sql/__init__.py

```diff
@@ -1,2 +0,0 @@
-import Orange.misc
-psycopg2 = Orange.misc.import_late_warning("psycopg2")
```

Comment:
Since this change, `import_late_warning` is used nowhere.
We could test if at least one of the backends is available and warn if none.